src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -1239,6 +1239,42 @@ def test_sql_db_long_term_retention(
             checks=[NoneCheck()])
 
 
+class SqlServerDbGeoRestoreScenarioTest(ScenarioTest):
+    def test_sql_db_geo_restore(
+            self):
+        self.kwargs.update({
+            'rg': 'rebeccaxu-test',
+            'loc': 'eastus',
+            'server_name': 'rebeccaxu-eastus-svr',
+            'database_name': 'cli-test-hs',
```

Comment:
Is it possible to create tmp resources for test instead of using existing resources? Others may not be able to rerun this test lively with fixed existing resources

Reply:
@calvinhzy had voiced this concern as well, I've added @record_only() to resolve the issue

Reply:
Unfortunately it usually takes a few hours for the geo-redundant backup to propagate, so if we made a new db every time the resources needed to perform the restore wouldn't be available.